001-devcycle.conf.template

```diff
@@ -1,25 +1,11 @@
-WSGIPythonPath usr/local/devcycle
```

Comment:
I belive it is recommended to run it as a daemon.  There was documentation on the [django page](https://docs.djangoproject.com/en/1.9/howto/deployment/wsgi/modwsgi/) if we want to update our config to that.